<launch>
   <param name="use_sim_time" value="true"/>
   <rosparam  command="load" file="$(find test_slam)/config/sys_param.yaml" />

   <node name="test_slam_node" pkg="test_slam"  type="test_slam_node" output="screen" >
       <remap from="scan" to="scan" />
   </node>
</launch>
